src/api/app/controllers/webui/project_controller.rb

```diff
@@ -698,12 +698,16 @@ def add_maintained_project_dialog
 
   def add_maintained_project
     authorize @project, :update?
-    @project.maintained_projects.new(project: @maintained_project)
-    if @project.save
+
+    @maintained_project = params[:maintained_project]
+    maintained_project = Project.where(name: @maintained_project)
+    if maintained_project.exists?
+      @project.maintained_projects.create!(project: maintained_project.first)
```

Comment:
ok, but is that consistent with `remove_maintained_project`? Because in that function we are redirecting back in case that `destroy` fails. Maybe we should change `remove_maintained_project` then. :confused: